src/frontend/src/routes/sitemap.xml/+server.ts

```diff
@@ -4,6 +4,7 @@ const staticPages: string[] = [];
 
 export const prerender = true;
 
+// eslint-disable-next-line require-await
```

Comment:
I did't quite understand this exception, why is it needed? because entire function is not awaitable?

Reply:
Because there is nothing to await within the function I think but given that's not really important here, I mean that's the root that generate the sitemap, I spared myself to dig deeper 😅. Works for you?

Reply:
yes yes, it was really a doubt, I was not understanding, thank you for the explanation!